src/main/java/com/test/ai/controller/HealthController.java

```diff
@@ -0,0 +1,15 @@
+package com.test.ai.controller;
+
+import com.test.ai.model.HealthModel;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthController {
+    @GetMapping("/health")
+    public HealthModel getHealth() {
+        HealthModel healthModel = new HealthModel();
+        healthModel.setStatus("UP");
```

Comment:
@coderabbitai - Our standard is to return a error code whenever the service is down instead of just saying down. Switch this to a ResponseEntity